genesis/engine/solvers/rigid/rigid_solver_decomp.py

```diff
@@ -4253,7 +4272,8 @@ def func_COM_links(
 
             i_r = links_info.root_idx[I_l]
             links_state.mass_sum[i_r, i_b] += mass
-            links_state.COM[i_r, i_b] += mass * links_state.i_pos[i_l, i_b]
+            links_state.link_COM[i_l, i_b] = links_state.i_pos[i_l, i_b]
```

Comment:
I don't think storing `link_COM` is useful because it is basically `i_pos`. Having multiple aliases for the same quantity is a bad practice and not memory friendly.